samples/csharp_dotnetcore/21.luis-with-appinsights/Middleware/Telemetry/LuisTelemetryConstants.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.BotBuilderSamples
+{
+    /// <summary>
+    /// The Application Insights property names that we're logging.
+    /// </summary>
+    public static class LuisTelemetryConstants
```

Comment:
Can you switch to PascalCasing for all the values? Then it will be consistent with `TelemetryConstants.cs` and othe Solutions [LuisTelemetryConstants.cs](https://github.com/Microsoft/AI/blob/master/templates/Enterprise-Template/src/csharp/EnterpriseBotSample/Middleware/Telemetry/LuisTelemetryConstants.cs
)

Reply:
Adding @benbrown.
Cool - Ben and I were having a discussion about the casing -- we're not inline with what appinsights uses, but will adopt your casing..

Reply:
One thing to note: `TelemetryLuisRecognizer` having the overload with the `DialogContext`, it introduces a new dependency.  For non-dialog scenarios, kind of a drag.